features/ALS016_Alignment-horizontal-segment-geometric-continuity.feature

```diff
@@ -0,0 +1,32 @@
+@industry-practice
+@ALS
+@version1
+
+Feature: ALS016 - Alignment horizontal segment geometric continuity
+
+  The rule verifies that there is geometric continuity between segments in an IfcCompositeCurve.
+  The calculated end position and tangent vector of segment `n` is compared to the provided placement of segment `n + 1`.
+  A warning is emitted if the calculated difference is greater than the applicable tolerance.
+  The tolerance for positional continuity is taken from the precision of the applicable geometric context.
+  The tolerance for tangential continuity is taken from the precision of the applicable geometric context and
+  adjusted based on the length of the alignment segment.
+
+Background:
+
+  Given A model with Schema "IFC4.3"
+  Given An IfcAlignment
+  Given Its attribute Representation
+  Given Its attribute Representations
+  Given RepresentationType = 'Curve2D'
+  Given All referenced instances
+  Given Its Entity Type is 'IfcCompositeCurve'
+  Given Its attribute Segments 
+  Given The values grouped pairwise
```

Comment:
```suggestion
  Given The values grouped pairwise at depth 1
```
I've ran this locally and it resolved the failing CI/CD. 
Without `at depth 1` we'd be executing`itertools.pairwise` for each segment individually instead of grouped together.
![image](https://github.com/user-attachments/assets/277e8f7a-3b97-4648-b9ad-f5cafcc0cb30)

Reply:
Agree, added `at depth 1`